saw-core-coq/src/SAWCoreCoq/Term.hs

```diff
@@ -456,7 +469,13 @@ flatTermFToExpr tf = -- traceFTermF "flatTermFToExpr" tf $
       return $ Coq.List elems
     StringLit s -> pure (Coq.Scope (Coq.StringLit (Text.unpack s)) "string")
 
-    ExtCns ec -> translateConstant (Name (ecVarIndex ec) (ecName ec))
+    -- ExtCns ec -> translateConstant (Name (ecVarIndex ec) (ecName ec))
```

Comment:
Can this commented-out line be deleted?

Reply:
Fixed in d82eb83ff82bca3fe43a3731f8e5ac7f34085779.